app/models/csv_files/csv_file.rb

```diff
@@ -0,0 +1,147 @@
+class CsvFile < ActiveRecord::Base
+  attr_accessor :upload
+
+  STI = %W(WeamsCsvFile)
```

Comment:
There are two STI, one in the csv_file_controller that was a leftover from the older version that is no longer needed (was a workaround for satisfy specs and  brakeman warnings with Single Table Inheritance), I will remove that. The other is required for validation and selecting and associated table containing the processed data of a csv. There will only be one.